### ISSUE TITLE
Prometheus plugin: expose Kubernetes annotations as tags

### ISSUE THREAD

**Issue by neographikal** [influxdata/telegraf#8793]
## Feature Request

It would be nice to have the option to also import and expose the Kubernetes annotations for discovered containers as tags. 

### Proposal:
Expose annotations as tags

### Current behavior:
-
### Desired behavior:
Expose annotations as tags

### Use case: 
This would enable us to tag containers with our Terraform deploys and filter the data in Elastic based on the annotations.

**Comment by powersj** [comment]
Hi,

Sorry, no one has gotten back to you yet. Is this something you are still interested in? If so, could you provide an example of what the annotations look like and what you might be expecting in telegraf?

Thanks

**Comment by ict-one-nl** [comment]
Yes I would be interested in this, we annotate our containers with metadata (like the business for which it is running). It would be nice to be able to collect all of this in one go and be able to report over it :)
